XXX: Only needed on powerpc, seems not to affect amd64 or loongson.

--- src/plugins/ario-plugin-info.c.orig	Fri May  3 22:28:48 2013
+++ src/plugins/ario-plugin-info.c	Fri May  3 22:29:10 2013
@@ -27,7 +27,7 @@
 
 #include <string.h>
 #include <glib/gi18n.h>
-#include <glib/gkeyfile.h>
+#include <glib.h>
 
 #include "plugins/ario-plugin-info-priv.h"
 #include "ario-debug.h"
